### DISCUSSION TITLE
DefaultLinkToExistingAccess

### DISCUSSION THREAD

**Discussion by 216rick** [pnp/powershell#173] [Q&A] [answered]
Is there an equivalent to SPOSite DefaultLinkToExistingAccess in PnP.PowerShell? 

DefaultSharingLinkType has options for everything except "Users with existing access".

**Comment by erwinvanhunen** [comment, marked as answer]
it wasn't there, but it will be there in release 1.2.7-nightly as of tomorrow. We just added it to both `Set-PnPSite` and `Set-PnPTenantSite` (the latter is effectively the PnP version of `Set-SPOSite`, whereas `Get-PnPTenantSite` the PnP equivalent is of `Get-SPOSite`)

**Reply by 216rick** [reply]
Excellent! Thank you!
